lrspline/__init__.py

```diff
@@ -593,8 +622,18 @@ def corners(self):
     def clone(self):
         return LRSplineVolume(self.w.copy())
 
+    def insert_knot(self, new_knots, direction, multiplicity=1):
+        new_knots = _ensure_listlike(new_knots)
+        direction = _check_direction(direction, self.pardim)
+        for k in new_knots:
+            start = list(self.start())
+            end = list(self.end())
+            start[direction] = end[direction] = k
+            mr = raw.MeshRectangle(start[0], start[1], start[2], end[0], end[1], end[2], multiplicity)
+            self.insert(mr)
+
     def insert(self, mr):
-        self.w.insert_line(mr.w.copy())
+        self.w.insert_line(mr.copy())
```

Comment:
This is a no-no. The `LRSplineVolume.insert` method should be usable on the Python side without having to know that its argument is some sort of wrapper object. Instead I've changed the `insert_knot` method to call into the raw objects directly.